src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createPromise(message) {
+  return new Promise((resolve, reject) => {
+    if (!message) {
+      resolve('Promise was resolved!');
+    } else {
+      setTimeout(() => reject(message), 3000);
+    }
+  });
+}
+
+function handleLogoClick() {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add('message');
+
+  const promise1 = createPromise();
+  const promise2 = createPromise('Promise was rejected!');
```

Comment:
You are creating two promises here, but `promise1` will always resolve immediately, and `promise2` will always reject after 3 seconds. This might not be what you intended. If you want to simulate a real-world scenario where a promise might either resolve or reject, you could consider using `Math.random()` to randomly determine whether the promise should resolve or reject.